veml6070/veml6070.go

```diff
@@ -0,0 +1,138 @@
+// Package bmp180 provides a driver for the BMP180 digital pressure sensor
```

Comment:
References to BMP180 digital pressure sensor

Reply:
Uh thanks for spotting my copy&paste error :+1: